CONTRIBUTING.md

```diff
@@ -1,37 +1,131 @@
-# How to contribute
 
-First of all, thank you for taking the time to contribute to this project. We've tried to make a stable project and try to fix bugs and add new features continuously. You can help us do more.
+## Table of Contents
+<!-- vim-markdown-toc GFM -->
 
-Before you start, read the **README.md** file for info on the project and how to set it up.
+* [How to contribute?](#how-to-contribute)
+* [Finding first issue](#finding-first-issue)
+* [Reporting issues](#reporting-issues)
+* [Creating commits](#creating-commits)
+* [Pull Requests](#pull-requests)
+* [Creating branch](#creating-branch)
+* [Tests](#tests)
 
-## Getting started
+<!-- vim-markdown-toc -->
+## How to contribute
 
-### Check out the roadmap
+First of all, thank you for taking the time to contribute to this project. :tada::tada::tada:
+We've tried to make a stable project and try to fix bugs and add new features continuously. You can help us do more.
 
-We have some functionalities in mind and we have issued them and there is a *milestone* label available on the issue. If there is a bug or a feature that is not listed in the **issues** page or there is no one assigned to the issue, feel free to fix/add it! Although it's better to discuss it in the issue or create a new issue for it so there is no conflicting code.
+Before you start, read the **[README.md](/README.md)** file for info on the project and how to set it up.
 
-### Filing issues
+## Finding first issue
+Go to [issues tab](https://github.com/LiskHQ/lisk-hub/issues) and look for 
 
-Before starting work on a larger idea not discussed in an issue we recommend starting one to iron out your approach to implementation. PRs with conflicting ideas regarding architecture or other aspects of the project may be rejected. We appreciate any and all ideas you contribute providing they're discussed in a respectful and constructive manner.
+*✏️ `good first issue`* - it is a label for tasks that are perfect for people who want to start with lisk-hub.
 
-Issues created that are not relevant to this project will be closed immediately - this is purely for efficiency as we don't have time to address and or move all of them to their correct place.
+## Reporting issues
 
-### Writing some code!
+ 1. Check if problem already exist in [issues tab](https://github.com/LiskHQ/lisk-hub/issues)
+ 2. If not, create a new [issue](https://github.com/LiskHQ/lisk-hub/issues/new)
+ 3. Fill our template:
+```
+Expected behaviour
 
-Contributing to a project on Github is pretty straight forward. If this is you're first time, these are the steps you should take.
+Actual behaviour
```

Comment:
I would add here the acctual markdown needed to be c&p into the issue field: 

```
### Expected behaviour
```

Reply:
Or just link to the real template https://github.com/LiskHQ/lisk-hub/blob/development/.github/issue_template.md so that any changes need to be made just in one place